bouncer/blacklist/views.py

```diff
@@ -14,30 +15,29 @@ class RequestView(View):
     template = loader.get_template("blacklist/request.html")
 
     def get(self, request, *args, **kwargs):
-        query = dict(self.request.GET.lists())
+        query = namedtuple("query", self.request.GET.keys())(**self.request.GET)
```

Comment:
It would be better to use [`getlist`](https://docs.djangoproject.com/en/2.0/ref/request-response/#django.http.QueryDict.getlist) function for every key we need. This defaults to an empty list `[]` if the key does not exist, so it will greatly simplify the code.